rules.bzl

```diff
@@ -96,11 +104,18 @@ def _create_symlinks(ctx, ndk_path, clang_directory, sysroot_directory):
         repo_relative_path = str(p).replace(ndk_path, "")
         ctx.symlink(p, repo_relative_path)
 
-android_ndk_repository = repository_rule(
+_android_ndk_repository = repository_rule(
     implementation = _android_ndk_repository_impl,
     attrs = {
         "path": attr.string(),
         "api_level": attr.int(),
     },
     local = True,
 )
+
+def android_ndk_repository(name, **kwargs):
+    _android_ndk_repository(
+        name = name,
+        **kwargs
+    )
+    native.register_toolchains("@androidndk//:all")
```

Comment:
can this be:
`native.register_toolchains("@%s//:all" % name)`

on the chance that the user names the the repo something other than `androidndk`

Reply:
Whoops, it absolutely was supposed to, that's why I captured the `name` parameter. Fixed.